rally-domain/src/main/groovy/yakworks/rally/orgs/OrgMemberService.groovy

```diff
@@ -25,10 +25,26 @@ import yakworks.rally.orgs.model.OrgType
 @Slf4j
 @CompileStatic
 class OrgMemberService {
+    private static final List<String> VALID_ORG_MEMBERS = ["Branch", "Division", "Business", "Sales", "Region", "Factory"]
```

Comment:
why can't we use OrgMemberRepo.validateMembers, it excludes custAccount, customer and company.
I prefer if we can exclude rater then specify valid org members (in case we want to add more)

If OrgMemberRepo.validateMembers is not used then remove it.